sql/moz-fx-data-shared-prod/telemetry/nondesktop_clients_last_seen_v1/view.sql

```diff
@@ -239,7 +239,7 @@ unioned AS (
     locale,
     NULL AS distribution_id,
     app_display_version AS app_version,
-    mozfun.norm.product_info(app_name, normalized_os) AS pinfo,
+    udf.product_info_with_baseline(app_name, normalized_os) AS pinfo,
```

Comment:
This results in all the "Baseline" apps showing up in the smoot tables as "Baseline" as well.